web_responsive/models/res_users.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    chatter_position = fields.Selection([
+        ('normal', 'Normal'),
+        ('sided', 'Sided'),
+    ], string="Chatter Position", default='normal')
+
```

Comment:
Nice to see this settings.

But don't you think it can have side effect with some module like 

https://github.com/OCA/web/tree/10.0/web_sheet_full_width_selective

Do you think it possible whatever user setting here, the chatter can go normal position if not enough width space ?

May be it works as it ?

Thanks s lot

Reply:
That's how it works actually.